### DISCUSSION TITLE
Documenting old tool chains.

### DISCUSSION THREAD

**Discussion by leamas** [opencpn-manuals/main#13] [General]
A simple question: is there any reason to document older tools than those we recommend and actually use these days?

On the Linux/Windows side we have dropped the mingw builds. Perhaps we should drop the corresponding documentation as well, it's very outdated and probably not used. Why spend time on it?

Same questions on the WIndows side for anything older than Visual Studio 2017. Why should we maintain documents for say 2015 (or even worse. 2013). They will not be correct, and we cannot maintain them since basically no-one have these tools installed.

etc...

Thoughts?

**Comment by leamas** [comment]
@rasbats @rgleason : ^

**Reply by rgleason** [reply]
Alec, I agree with you.

**Comment by Rasbats** [comment]
Agree. Encourage use of latest (almost) tools. Checking workflows with these deprecated tools would be a pain.

**Reply by leamas** [reply]
OK. I will remove the mingw stuff.

**Reply by leamas** [reply]
@rgleason is working with the windows stuff, for sure.

**Reply by leamas** [reply]
Mingw removed in https://github.com/opencpn-manuals/development/commit/4b189486774e1da9cc986526c0b70d0c25ff7431. Time to rebase...
